src/_Core.scss

```diff
@@ -203,6 +203,10 @@ body {
   &.explore-nav {
     margin-left: -24px;
     height: unset;
+    .MuiTabScrollButton-root .MuiSvgIcon-root {
```

Comment:
You might be able to add a custom css class (in case MUI decides to rename their classes in the future).
Like this:  https://github.com/Sage-Bionetworks/Synapse-React-Client/pull/1246/files#diff-71e15e034ccd52858f944a7e94be6963cbabff2612660573f8ceaab9609a85b0R6